buildSrc/subprojects/kotlin-dsl/src/main/kotlin/codegen/GenerateKotlinDependencyExtensions.kt

```diff
@@ -22,13 +22,12 @@ import org.gradle.api.tasks.Input
 import org.gradle.api.tasks.OutputFile
 import org.gradle.api.tasks.TaskAction
 
-import java.io.File
-
 
+@Suppress("unused")
 open class GenerateKotlinDependencyExtensions : DefaultTask() {
 
     @get:OutputFile
-    var outputFile: File? = null
+    val outputFile = project.objects.fileProperty()
 
     @get:Input
     var embeddedKotlinVersion: String? = null
```

Comment:
:x: Let's use `Property<String>` for this property and the one below

Reply:
Would you care to clarify why, @eskatos?

Reply:
For removing the nullability & `var`, also for uniformity and dogfooding.

BTW now it could be just:
```kotlin
@get:Input
abstract val embeddedKotlinVersion: Property<String>
```

Reply:
I assume you ment `lateinit` instead of `abstract`. Makes sense 👍

Reply:
I meant `abstract`. Gradle will wire the property automatically.

Reply:
Oh, really? I missed that. Nice 👍

Reply:
Done.